contents/newsletter/talk-to-users.md

```diff
@@ -0,0 +1,141 @@
+---
+title: 8 things engineers should know about talking to users
+date: 2024-04-22
+author:
+  - ian-vanagas
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/newsletter/beyond-10x-engineer/super-hog.png
+featuredImageType: full
+tags:
+  - Engineering
+  - Product engineers
+crosspost:
+  - Product engineers
+  - Blog
+---
+
+Talk to users? Isn't that a job for founders, product managers, and salespeople? Why would an engineer ever want or need to do it?
+
+In his book *[Modern Software Engineering](https://www.davefarley.net/?p=352)*, Dave Farley writes that mastering software engineering requires becoming an expert in learning. When you prioritize learning, talking to users becomes a critical part of an engineer's toolkit.
+
+To explain further why this is the case, and to help you get started, we wrote this guide based on what we've learned talking to users as engineers ourselves at PostHog.
```

Comment:
how about something like...

That's why you should be talking to users. But what about how? Here are eight essential tips to make sure you get the most out of those conversations.